### PR TITLE
issue_120: Allow the backup property to be specified as true

### PR DESCRIPTION
**Pull request by MarkGibbons**
Version 2.1.1

Fix issue #120 by allowing the backup property to have a value of true.

[List any existing issues this PR resolves]

#120

- [ ] All tests pass.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable

---
This change is [Reviewable](https://reviewable.io/reviews/sous-chefs/line/122)

### CODE REVIEW

File: resources/add_to_list.rb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-property :backup, [true, false], default: false
+property :backup, [true, false, Integer], default: false
```
Comment (damacus) [inline resources/add_to_list.rb:1]: You're going to have to talk me through now a boolean needs an Integer value

Comment (MarkGibbons) [reply resources/add_to_list.rb:1]: Backup gets passed to the file resource.  File only accepts false or Integer.  I didn't read the file resource doc closely enough and thought it was a boolean.  If I hadn't advertised this as allowing true I'd probably go back to matching the file resource and only allow false or Integer.  The design of file is nuts though.  Allowing an integer and false, but not true is just wrong.

Line historically has not taken backups.   So I set the default to false to match past behavior.

**Comment by MarkGibbons** [comment]
I'll see if I can figure out where the backup files are stored to make the tests run cleanly.

File: resources/add_to_list.rb (hunk 1 above)
Comment (MarkGibbons) [reply resources/add_to_list.rb:1]: I deleted the checks that the backup files were created.  We don't need to check that the file resource creates backups.  The tests should pass now.

**Review by damacus — COMMENTED** [review]
> Reviewed 14 of 20 files at r1, 1 of 6 files at r2.
> *[Reviewable](https://reviewable.io/reviews/sous-chefs/line/122)* status: all files reviewed, 1 unresolved discussion (waiting on @damacus)

**Comment by damacus** [comment]
___
*[resources/add_to_list.rb, line 1 at r1](https://reviewable.io/reviews/sous-chefs/line/122#-LO8kKruShENlZ0-RHwb-r1-1:-LOK9Anq4lUCoqCB9VQx:bn6y38h) ([raw file](https://github.com/sous-chefs/line/blob/7aabe86bcab5224d786cca8ba66dc8d56301c0d3/resources/add_to_list.rb#L1)):*
<details><summary><i>Previously, MarkGibbons (Mark Gibbons ) wrote…</i></summary><blockquote>

I deleted the checks that the backup files were created.  We don't need to check that the file resource creates backups.  The tests should pass now.
</blockquote></details>

Next major release, remove `true` ?

**Review by damacus — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/sous-chefs/line/122)* status: :shipit: complete! all files reviewed, all discussions resolved

**Comment by damacus** [comment]
:lgtm:

**Review by damacus — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/sous-chefs/line/122)* status: :shipit: complete! all files reviewed, all discussions resolved

**Pull request merged**